### ISSUE TITLE
Hiding console window on Windows build

### ISSUE THREAD

**Issue by parkersprouse** [deepnight/redistHelper#5]
Hello!

I'm not very familiar with building C code, so redistHelper has been a life saver for my Heaps projects, so thank you for creating it!

One thing that I would like to ask about though is the console window that pops open with the game when you launch a project built on Windows. I know you can call `hl.UI.closeConsole()` to close it after the game launches, but that still results in the prompt briefly flashing when launching the game, and I personally dislike that.

Are there any changes that can be made to redistHelper to remove the console window entirely? Any flags that can be added to the Windows build command that can accomplish this? I'd be willing to fork / clone the repo and do it myself, I'm more asking for some direction if you didn't have the time or weren't personally interested in doing it yourself.

Thank you for any help!

**Comment by deepnight** [comment]
The issue is related to Hashlink itself, and I agree that an option to not show the console at all would be great :)

You can write an issue there: https://github.com/HaxeFoundation/hashlink

**Comment by LeGrimoire** [comment]
Hi ! 
This seems to have solve the issue : https://github.com/HaxeFoundation/hashlink/issues/519#issuecomment-1072914590
> `editbin.exe /subsystem:windows hl.exe`, where `editbin` is a part of Visual Studio

**Comment by deepnight** [comment]
True, but it forces the user to install the whole Visual Studio thing just to change this app flag 🤔  I'm still hoping for an actual fix, as it used to work before some undetermined update.

**Comment by LeGrimoire** [comment]
What about updating the bundled hl.exe ?
